### PR TITLE
add Theranos Coin (LIZ)

### PR DESCRIPTION
**Pull request by drinksbydrew**
**Description**
Add Theranos Coin (LIZ) to the token list.

I approved `0xe3a90926133fef3c58514fA1aAeF75E0998aedD5` to spend [LIZ on kovan](https://kovan.etherscan.io/token/0x47Aa2a683b88E23c4d76f91Aa4181A59d0E4FBfb).

### CODE REVIEW

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -1023,6 +1023,50 @@
...
+      "symbol": "LIZ",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/LIZ.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 42,
+      "address": "0x47Aa2a683b88E23c4d76f91Aa4181A59d0E4FBfb",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1051]: Can you verify this contract in Etherscan

Comment (elenadimitrova) [inline optimism.tokenlist.json:1051]: Can you transfer some amount to `0xe3a90926133fef3c58514fA1aAeF75E0998aedD5` on kovan for testing pls

Comment (drinksbydrew) [reply optimism.tokenlist.json:1051]: yes, is this what you meant? https://kovan.etherscan.io/address/0x47aa2a683b88e23c4d76f91aa4181a59d0e4fbfb#code

Comment (drinksbydrew) [reply optimism.tokenlist.json:1051]: yes, done: https://kovan.etherscan.io/tx/0xc29dd0b2f629c8807475e560146f7acc9125c2e641dceda117511d24c2c456d7

Comment (elenadimitrova) [reply optimism.tokenlist.json:1051]: yes thanks

**Review by elenadimitrova — APPROVED** [review]

**Comment by elenadimitrova** [comment]
Kovan test 
Deposit 1000 LIZ
https://kovan.etherscan.io/tx/0x8885e1d70c5a636d36f49c5dabd886ec9b93a4eb1d0a5b3314a3b32744b29e7d
https://kovan-optimistic.etherscan.io/tx/0x4426d0915583531763619ee26f51db713a98ba93d2bd6ab3f2e8ca7ead70a221

Withdraw 500 LIZ
https://kovan-optimistic.etherscan.io/tx/0x23b1a73b0ae18b16919a0995193901eb1b993aebbbf27d485d6f9e02cb9ac95e
https://kovan.etherscan.io/tx/0x9e0b8fabb4af14c85726303ef891b07956a30fe71b13a1cf669773a051f4ea15

**Pull request merged**